src/lib/Libutil/misc_utils.c

```diff
@@ -1782,3 +1782,59 @@ perf_stat_stop(char *instance)
 
 	return (stat_summary);
 }
+
+/**
+ * @brief
+ *	creates an empty file in /tmp/ and saves timestamp of that file
+ *
+ * @param[in] - void
+ *
+ * @return - void
+ */
+void
+create_query_file(void)
+{
+	char filename[30];
+	char command[30];
+	snprintf(filename, 30, "/tmp/last_query_%d", getuid());
+#ifdef WIN32
+	snprintf(command, 30, "type nul > %d", filename);
+#else
+	snprintf(command, 30, "touch %s", filename);
+#endif
+	system(command);
+}
+
+/**
+ * @brief
+ *	stats te information of the empty file created in /tmp/ to decide 
+ *  whether to add sleep for .2 secons or not
+ * 
+ * @param[in] - void
+ *
+ * @return - void
+ */
+void
+check_last_query(void)
```

Comment:
delay_query() may be a more descriptive name and would allow us to delay different amounts of time based on different criteria (e.g. machine load) in the future.